### PR TITLE
(maint) Add maintainers section to README.md

### PR DESCRIPTION
**Pull request by kbarber**
I also fixed the old puppetlabs.com address for the docs link.

Signed-off-by: Ken Barber <ken@bob.sh>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
-[docs]: https://docs.puppetlabs.com/puppetdb/latest
+[docs]: https://docs.puppet.com/puppetdb/latest
 
 PuppetDB is the fast, scalable, and reliable data warehouse for Puppet. It caches data generated by Puppet, and gives you advanced features at awesome speed with a powerful API.
 
 For documentation on this product, consult the [latest documentation][docs].
+
+## Maintenance
+
+Maintainers: Ryan Senior <ryan.senior@puppet.com>, Wyatt Alt <wyatt@puppet.com>, Rob Browning <rlb@puppet.com>, AJ Roetker <andrew.roetker@puppet.com>, Russell Mull <russell.mull@puppet.com>, Ken Barber <ken@puppet.com>
```
Comment (ajroetker) [inline README.md]: What about putting our alias here? puppetdb-team@puppet.com

Comment (kbarber) [inline README.md]: I think there was some debate about putting aliases in an internal email thread at the moment, we can wait to see where that goes or chime in I think ... also, I'm not sure how public-ready the alias is, we might need to ensure we get moderation for example (as a moderator of puppet-users, this becomes a thing eventually).

**Comment by wkalt** [comment]
@kbarber should this be against stable?

File: README.md (hunk 1 above)
Comment (senior) [inline README.md]: @lutter This doesn't seem like the right thing for a Puppet product vs. a shared library (public or internal). In product terms, people working on it is more fluid. Is puppet-users or some other generic public forum a better address for this?

Comment (lutter) [inline README.md]: Agreed - this is a bad fit for projects with well-established practices. How about you add a sentence to this section along the lines "For questions/problems/dev help email <our public mailing list>" and then as maintainers list one or two of the people working on PuppetDB. The reason for listing people is to give others a way to get in touch with a human if they can't get attention through other means. In that sense, the maintainer duties are mostly around redirecting and connecting people with the right devs for whatever their problems are.

Comment (senior) [inline README.md]: I talked with @lutter a bit on this. It seems like the best approach here is to point people to puppet-users and puppet-dev (with a short description of what discussions go where).

**Comment by senior** [comment]
Handled in https://github.com/puppetlabs/puppetdb/pull/1959

**Pull request closed without merging**